conf/nd96isr_mi300x_v5.conf

```diff
@@ -48,6 +48,7 @@
 ####
   * || check_gpu_count 8 rocm
   * || check_gpu_bw 48 44
+  * || check_rccl_allreduce 304 1 16G
```

Comment:
How tight do we need to be to the observed value? Does 304 have some buffer added?

Reply:
We have about a 5% buffer. 320 was what I observed normally. This PR was also tested at scale on an MI300 cluster.